src/main/java/org/opentripplanner/raptor/rangeraptor/path/DestinationArrivalPaths.java

```diff
@@ -55,6 +55,8 @@ public class DestinationArrivalPaths<T extends RaptorTripSchedule> {
   private boolean reachedCurrentRound = false;
   private int iterationDepartureTime = -1;
 
+  // TODO: 2023-05-22 via pass through: we need to inject expected c2 value here
```

Comment:
This comment is irrelevant. There's a special implementation for transitCalculate which validates c2 values.